extensions/wikia/UserLogin/js/spec/UserBaseAjaxForm.spec.js

```diff
@@ -0,0 +1,54 @@
+/* global UserBaseAjaxForm */
+
+describe('Base class for User Ajax Forms', function () {
+	'use strict';
+
+	var instance, $el, options,
+		proto = UserBaseAjaxForm.prototype,
+		$userNameInput = $('<input type="text" name="username">'),
```

Comment:
Ha! You found some code which wasn't touched by me :)
I need to investigate what happened there.